dptb/postprocess/unified/calculator.py

```
@@ -136,13 +150,35 @@ def model_forward(self, atomic_data: dict) -> dict:
+            orb_names = []
+            for sym in self.model.idp.type_names:
+                with open(f"./temp_{sym}.orb", "w") as f:
+                    f.write(self.model.orbital_files_content[sym])
+                orb_names.append(f"temp_{sym}.orb")
+            atomic_data = compute_overlap(atomic_data, self.model.idp, "./", orb_names)
+            # remove the temporary files
+            for orb_name in orb_names:
+                os.remove(orb_name)
+        elif self.overlap:
+            if self.model.method == "e3tb":
+                log.warning("The overlap inferenced from model is not stable in singular basis, please ensure this is what you want.")
```

Ruff (0.15.0)

[error] Undefined name `log`

(F821)